src/DssSpellCollateralOnboarding.sol

```diff
@@ -31,60 +35,93 @@ contract DssSpellCollateralOnboardingAction {
     // A table of rates can be found at
     //    https://ipfs.io/ipfs/QmTRiQ3GqjCiRhh1ojzKzgScmSsiwQPLyjhgYSxZASQekj
     //
+    uint256 constant THREE_POINT_FIVE_PCT = 1000000001090862085746321732;
 
     // --- Math ---
+    uint256 constant THOUSAND   = 10 ** 3;
+    uint256 constant MILLION    = 10 ** 6;
+
+    // --- PRE-REQUISITE GENERAL DEPLOYS ---
+    address constant CDP_REGISTRY              = 0xBe0274664Ca7A68d6b5dF826FB3CcB7c620bADF3;
+    address constant PROXY_ACTIONS_CROPPER     = 0xa2f69F8B9B341CFE9BfBb3aaB5fe116C89C95bAF;
+    address constant PROXY_ACTIONS_END_CROPPER = 0xAa61752a5Abf86A527A09546F23FE8bCB8fAB2C4;
+    address constant CROPPER                   = 0x8377CD01a5834a6EaD3b7efb482f678f2092b77e;
```

Comment:
As discussed (and so we don't forget) - the rely/deny events in Cropper are logging msg.sender so we probably want to fix that and redeploy.

Reply:
done